src/game/rendering_graph_node.c

```diff
@@ -1039,6 +1039,8 @@ void geo_process_object(struct Object *node) {
     if (node->header.gfx.areaIndex == gCurGraphNodeRoot->areaIndex) {
         s32 isInvisible = (node->header.gfx.node.flags & GRAPH_RENDER_INVISIBLE);
         s32 noThrowMatrix = (node->header.gfx.throwMatrix == NULL);
+        // Maintain throw matrix pointer if the game is paused as it won't be updated.
+        Mat4 *oldThrowMatrix = (sCurrPlayMode == PLAY_MODE_PAUSED) ? node->header.gfx.throwMatrix : NULL;
```

Comment:
Not sure about this one, wouldn't this cause issues with cutscenes or other scenarios that don't update an object that frame? Not sure of an elegant solution for this though, so gonna keep that non-blocking.

Reply:
why would this cause issues? if the game isn't paused, then oldThrowMatrix will be NULL, which will lead to the exact same behavior as vanilla (throwMatrix being set to NULL).

Reply:
Issues isn't the right word, moreso that it would still reset the throw matrix during cutscenes.

Reply:
well yeah but itd still do that in vanilla so it's not a regression or anything. ive never heard of anyone having an issue with that before compared to how common the pause issue is so this pr isnt really addressing it, i imagine it would be easy to add later if needed